File: CapstoneProject/Assets/Scripts/Collider.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Collider : MonoBehaviour {

    public Inventory inventory;

	// Use this for initialization
	void Start () {
		
	}
	
	// Update is called once per frame
	void Update () {
		
	}
    void OnCollisionEnter(Collision coli) {
        if (coli.gameObject.tag == "Input")
        {
            inventory.Open(Inventory.Mode.AddMode);
        }
        else if (coli.gameObject.tag == "Output")
        {
            inventory.Open(Inventory.Mode.MinusMode);
        }
        else if (coli.gameObject.tag == "Inventory")
        {
            inventory.Open(Inventory.Mode.Open);
        }
    }
}

File: CapstoneProject/Assets/Scripts/Collider.js
#pragma strict

public var player : GameObject ;
public var inventory : GameObject;
public var delection : GameObject;
public var inserction : GameObject;

public var popup : AudioClip;

function Start ()
{

}

function Update ()
{
	
}

function OnCollisionEnter (coll:Collision)
{
	if(coll.gameObject.tag == "Input")
	{
		player.GetComponent(inven).re();
        inserction.SetActive(true);
		GetComponent.<AudioSource>().Play();
		
		print("삽입");
	}
	else if(coll.gameObject.tag == "Output")
	{	player.GetComponent(inven).re();
		delection.SetActive(true);
		GetComponent.<AudioSource>().Play();

		print("삭제");
	}
	else if(coll.gameObject.tag == "Inventory")
	{	player.GetComponent(inven).re();
		inventory.SetActive(true);
		GetComponent.<AudioSource>().Play();

		print("관리");
	}

}